ouroboros-consensus/ouroboros-consensus-mock/test/Test/ThreadNet/BFT.hs

```diff
@@ -44,11 +46,30 @@ tests = testGroup "BFT"
           , slotLengths = singletonSlotLengths (slotLengthFromSec 1)
           , initSeed = Seed {getSeed = (12659702313441544615,9326820694273232011,15820857683988100572,2201554969601311572,4716411940989238571)}
           }
-    ,
-      testProperty "simple convergence" $ \tc ->
+    , testProperty "Mock.applyChainTick is not a no-op" $
+        -- This repro failed via on a wip branch that included a fix for Issue
+        -- 1489 and but not for Issue 1559. PR 1562 fixed it. We're retaining
+        -- this as a regression test.
+        once $
+        let ncn = NumCoreNodes 3 in
+        prop_simple_bft_convergence (SecurityParam 5)
+        TestConfig
+          { numCoreNodes = ncn
+          , numSlots     = NumSlots 7
+          , nodeJoinPlan = NodeJoinPlan $ Map.fromList [(CoreNodeId 0, SlotNo 0),(CoreNodeId 1, SlotNo 2),(CoreNodeId 2, SlotNo 2)]
+          , nodeRestarts = noRestarts
+          , nodeTopology = meshNodeTopology ncn
+          , slotLengths  = defaultSlotLengths
+          , initSeed     = Seed (6358650144370660550,17563794202468751585,17692838336641672274,12649320068211251815,18441126729279419067)
+          }
+    , testProperty "simple convergence" $ \tc ->
+        -- TODO k > 1 as a workaround for Issue #1511.
+        --
         forAll (SecurityParam <$> elements [2 .. 10]) $ \k ->
         prop_simple_bft_convergence k tc
     ]
+  where
+    defaultSlotLengths = singletonSlotLengths $ slotLengthFromSec 1
```

Comment:
Does the repro depend on the exact slot length?

I'm inclined to use 20s here, because that's the default slot length for BFT.

Also, you could use `defaultSlotLengths` in the first test case too.

Reply:
I switched it to 20s. Done, thanks.